### ISSUE TITLE
runpodctl config command fails on linux

### ISSUE THREAD

**Issue by VitalikLev** [runpod/runpodctl#160]
Hello,

I am facing an issue configuring runpodctl on Ubuntu. When I run the command:

```runpodctl config --apiKey <my_api_key>```

I receive the following response:

```Runpod config file not found, please run `runpodctl config` to create it```

```Error: error saving config: Config File ".runpod.yaml" Not Found in "[/root/.runpod /root]"```

The command works fine on macOS, and I’m certain the API key is correct. It was working on Ubuntu as recently as September 6th 2024.

**Comment by DireLines** [comment]
Ok, gotcha. I will try to test that. Thanks for reporting it!

I know the API key works on mac, but try with a fresh API key just in case. Are you on the latest runpodctl version, v1.14.4?

**Comment by VitalikLev** [comment]
yes, I'm on the latest version of runpodctl. Api key is also fresh

**Comment by VitalikLev** [comment]
Upd: I ran `runpodctl config --apiKey <my_api_key>` on github runners(I tried to set linux and macos machines and it failed in both cases). What is interesting is that I was not able to reproduce it locally(despite using same api key and latest runpodctl version in all cases). 

`runpodctl config --apiKey=$RUNPOD_API_KEY || true` helped to fix this problem

**Comment by yoshi12u** [comment]
Hello,
I'm experiencing a similar issue when running runpodctl from within a RunPod pod. After executing:
```
runpodctl config --apiKey <my_api_key>
```
I got:
```
Configuration saved to file: /root/.runpod/config.toml
SSH key pair generated: /root/.runpod/ssh/RunPod-Key-Go (private), /root/.runpod/ssh/RunPod-Key-Go.pub (public)
Failed to add the SSH key: failed to get existing SSH keys: API error: Unauthorized
```

I have verified that the API key is valid and has the correct permissions. The same commands work as expected on macOS. I also tried generating a fresh API key, but the issue persists on RunPod pod.

Any advice would be greatly appreciated.
Thank you!

**Comment by aidando73** [comment]
I'm getting the same issue as @yoshi12u

**Comment by TimPietruskyRunPod** [comment]
Resolved in [v2.0.0](https://github.com/runpod/runpodctl/releases/tag/v2.0.0).

```bash
runpodctl doctor
```

The `doctor` command replaces `config` and properly creates the config directory and file on Linux (and all platforms). It guides you through API key setup and validates your configuration.
